src/components/common/TmBalance.vue

```diff
@@ -143,13 +189,14 @@
 </template>
 <script>
 import { bigFigureOrShortDecimals } from "scripts/num"
+import { removeLastCharacter } from "src/scripts/common"
```

Comment:
what makes this script necessary?

Reply:
that I am sending `fiatValues` from the API with the currency symbol (default `€`) at the very end

Reply:
seems like this could be improved in the API. maybe something like this would work:
```
denom: 'USD'
denomSymbol: '$'
```
...so that the frontend doesn't have to re-format things that we have control of in the API.

Reply:
🤔 so fiatValue would be an Object then. Yes, I think it is a pretty good idea actually